backend/environment.py

```diff
@@ -1,3 +1,12 @@
 import os
 
+from aws_cdk.core import RemovalPolicy
+
 ENV = os.environ.get("DEPLOY_ENV", "test")
+RESOURCE_REMOVAL_POLICY = os.environ.get("RESOURCE_REMOVAL_POLICY", "DESTROY")
+
+if RESOURCE_REMOVAL_POLICY.upper() == "RETAIN":
+    REMOVAL_POLICY = RemovalPolicy.RETAIN
```

Comment:
We should probably move this to the infrastructure directory, since it's only used there.

Reply:
Code moved to `infrastructure/removal_policy.py` file